plugins/restapi/v2/node.go

```diff
@@ -48,13 +58,27 @@ func info() (*infoResponse, error) {
 		Name:    deps.AppInfo.Name,
 		Version: deps.AppInfo.Version,
 		Status: nodeStatus{
-			IsHealthy:                deps.Tangle.IsNodeHealthy(),
-			LatestMilestoneTimestamp: latestMilestoneTimestamp,
-			LatestMilestoneIndex:     latestMilestoneIndex,
-			ConfirmedMilestoneIndex:  confirmedMilestoneIndex,
-			PruningIndex:             pruningIndex,
+			IsHealthy: deps.Tangle.IsNodeHealthy(),
+			LatestMilestone: milestoneInfoResponse{
+				Index:       latestMilestoneIndex,
+				Timestamp:   latestMilestoneTimestamp,
+				MilestoneID: latestMilestoneIDHex,
+			},
+			ConfirmedMilestone: milestoneInfoResponse{
+				Index:       confirmedMilestoneIndex,
+				Timestamp:   confirmedMilestoneTimestamp,
+				MilestoneID: confirmedMilestoneIDHex,
+			},
+			PruningIndex: pruningIndex,
+		},
+		Protocol: protocolResponse{
+			Version:       deps.ProtocolParameters.Version,
+			NetworkName:   deps.ProtocolParameters.NetworkName,
+			Bech32HRP:     string(deps.ProtocolParameters.Bech32HRP),
+			MinPowScore:   deps.ProtocolParameters.MinPowScore,
+			RentStructure: deps.ProtocolParameters.RentStructure,
+			TokenSupply:   iotago.EncodeUint64(deps.ProtocolParameters.TokenSupply),
 		},
-		Protocol: deps.ProtocolParameters,
```

Comment:
please revert this change, ProtocolParameters encode correctly to json.
Also remove `protocolResponse` type